homeassistant/components/broadlink/switch.py

```diff
@@ -121,6 +122,9 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     elif device.api.type == "SP2":
         switches = [BroadlinkSP2Switch(device)]
 
+    elif device.api.type == "BG1":
+        switches = [BroadlinkBG1Slot(device, slot) for slot in range(1, 3)]
+
```

Comment:
Move this if statement down one place, so that it is between SP4 and MP1.

Reply:
Done